libr/core/rvc.c

```diff
@@ -0,0 +1,115 @@
+#include <rvc.h>
+static bool copy_commits(const Rvc *repo, const char *dpath, const char *sname) {
+	char *path, *name, *spath;
+	spath = r_str_newf ("%s" R_SYS_DIR "branches" R_SYS_DIR "%s" R_SYS_DIR "commits", repo->path, sname);
```

Comment:
Define and assign in lone line

Reply:
I will declare and define spath on it's own beacause doing that with the others would cause the line to be 122 chars long